packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
-
-![alert group](./img/alert-group.png)
-
-### Modal alerts
-When a condition occurs that requires immediate action, a modal message dialog is recommended.
+An [Alert Group](documentation/react/components/alertgroup) component allows a set of alerts to be properly positioned and stacked as shown below.
 
-![modal notification](./img/modal-notification.png)
-
-For more information about designing modal messages see the [message dialog design guidelines](/design-guidelines/usage-and-behavior/modal).
+![alert group](./img/toast-notification-multiple.png)
 
 ## Content guidelines
 Message text should be succinct and clearly state what the issue or problem is. Embedded links should navigate the user to the location where they can take action.
 
-Alert notifications present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore we recommend that any message containing actions be descriptive of how the action can be performed in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
+Toast alerts present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore, we recommend that any message containing actions be descriptive of how the can navigate to perform the action in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
 
-![non-accessible toast notification](./img/notification-not-accessible.png)
+<img src="./img/Notification-content-not-accessible.png" alt="Non-accessible alert example" width="615">
 
 A more accessible version of this message would look like this:
 
-![non-accessible toast notification](./img/notification-accessible.png)
+<img src="./img/Notification-content-accessible.png" alt="Non-accessible alert example" width="615">
 
 While this is more verbose, it has the advantage of being usable for both sighted and non-sighted users by providing instructions to screen reader users about where they can go to complete the action.
 
+## Examples
+
+#### Toast alerts
+Toast alerts always stack in the upper right of a page.
+
+<img src="./img/toast-alerts-desktop.png" alt="toast alerts in a page" width="990">
+
+
+#### Inline alerts
+If an alert message applies globally to the content on a page, place the alert in the page header area just below the title.
+
+<img src="./img/Inline-alert-page-level.png" alt="global inline alert in a page" width="990">
```

Comment:
Convert to self-closing img tag by adding / before the closing bracket:
`<img src="./img/Inline-alert-page-level.png" alt="global inline alert in a page" width="990" />`